Modules/Core/Common/include/itkMacro.h

```diff
@@ -862,7 +862,7 @@ compilers.
 // clang-format off
 /** Set a decorated input. This defines the Set"name"() and a Set"name"Input() method */
 #define itkSetDecoratedInputMacro(name, type)                                                                       \
-  virtual void Set##name##Input(const SimpleDataObjectDecorator<type> * _arg)                                       \
+  virtual void Set## name## Input(const SimpleDataObjectDecorator<type> * _arg)                                       \
```

Comment:
Is this space after `##` expected and/or desired?

Reply:
reverted.  I don't know why this got auto-formatted as a change.  clang-format does not seem to care which way it is formatted.